examples/python/signed_distance_fields/main.py

```diff
@@ -79,21 +79,21 @@ def read_mesh(path: Path) -> Trimesh:
     return cast(Trimesh, mesh)
 
 
-@log_timing_decorator("global/voxel_sdf", rr.LogLevel.DEBUG)  # type: ignore[misc]
+@log_timing_decorator("global/voxel_sdf", "DEBUG")  # type: ignore[misc]
```

Comment:
here and in all other python example code: Why not use constants for the level strings?

Reply:
Truth? Because I am fed up with Python and so I threw my hands in the air and said "what's the difference - the errors are only caught at runtime anyway - at least a string literal is short"

Reply:
😢

Reply:
…also I thought I hade opened this PR in draft mode 😅

Reply:
I also don't know how to do it. `rr2.components.TextLogLevelType.INFO` doesn't work: `module 'rerun.components' has no attribute 'TextLogLevelType'`

Reply:
I'll fix it in a later PR